### PR TITLE
Update Segment with to be compatible with 11.0.00

### PR DESCRIPTION
**Pull request by arshiya-moengage**
https://moengagetrial.atlassian.net/browse/MOEN-9293

### CODE REVIEW

**Comment by umangmoe** [comment]
MoEngage SDK version update missed in `configuration.gradle`

File: example/build.gradle [hunk 1]
```diff
@@ -11,7 +11,7 @@ android {
   properties.load(file('../local.properties').newDataInputStream())
 
   defaultConfig {
-    applicationId "com.moengage.segment.example"
+    applicationId "com.moengage.sampleapp"
```
Comment (umangmoe) [inline example/build.gradle:14]: This change is not required. Leave the example as is.

**Comment by umangmoe** [comment]
Target and compile SDK version should be updated to 29

File: example/build.gradle (hunk 1 above)
Comment (arshiya-moengage) [reply example/build.gradle:14]: fixed

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
+### 5.3.00
```
Comment (umangmoe) [inline CHANGELOG.md:1]: should be 5.0.00

File: example/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -1,18 +1,18 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.moengage.segment.example">
+    package="com.moengage.sampleapp">
```
Comment (umangmoe) [inline example/src/main/AndroidManifest.xml:3]: fix package name

File: CHANGELOG.md (hunk 2 above)
Comment (arshiya-moengage) [reply CHANGELOG.md:1]: fixed

File: example/src/main/AndroidManifest.xml (hunk 3 above)
Comment (arshiya-moengage) [reply example/src/main/AndroidManifest.xml:3]: fixed

**Review by umangmoe — APPROVED** [review]

**Pull request merged**